FoodAPI/AppAPI/views.py

```diff
@@ -11,16 +11,26 @@
 
 
 
-
+# List rests and post single rest
 @csrf_exempt
 def restList(request):
     if request.method == 'GET':
         restObj = Rest.objects.all()
```

Comment:
Add delete function

Reply:
Thats a great idea and will be added in a future commit.